pkg/ebpf/tracer.go

```diff
@@ -308,6 +310,8 @@ func (t *Tracer) removeEntries(mp, tcpMp *bpflib.Map, entries []*ConnTuple) {
 	}
 
 	t.state.RemoveConnections(keys)
+
+	log.Infof("Removed %d entries in %+v", len(keys), time.Now().Sub(now))
```

Comment:
I also don't think we should print this every 30s. We'd have nearly 3k messages a day.